src/octoprint/templates/dialogs/files.jinja2

```diff
@@ -70,6 +70,19 @@
                 </div>
             </div>
         </form>
+        <div>
+            <b>Existing file</b>
+            <ul>
+            <li>Size: <span class="existing_size"></span></li>
```

Comment:
Do the classes actually mean anything? As far as I can see they are used as a way to reference the element in the jquery selector. Perhaps it would make more sense to use actual IDs?

Reply:
cool, will update that later today

Reply:
Tbh, given how the query is done, I'd prefer we leave it a class so as to not run into namespace issues.

`.existing_size` within `self.uploadExistsDialog` (so almost like it is now, I'd simply get rid of the `span` specification, could be another element at some point) is perfect for the use here.

If we use an id here, we'll need to look into name spacing, `existing_size` is too unspecific for a global site-wide id.

Reply:
No, the way it is now it's better, as it separates the value from the label (which btw also still needs to be made translatable, e.g. `{{ _('Size:') }} `).

Reply:
done